docs/account_system.md

```diff
@@ -0,0 +1,158 @@
+Account system
+==============
+
+The account management is homebrew, except the password hashing which is handled by [bcrypt](https://godoc.org/golang.org/x/crypto/bcrypt) (adaptative hashing algorithm).
```

Comment:
Do we want to be responsible for storing and managing credentials of our own?  Perhaps less risky might be to have users log in with a different SSO provider -- for instance, using Google or GitHub OAuth (or perhaps even [another wearable SSO source](https://dev.fitbit.com/build/reference/web-api/oauth2/) :) ).

Reply:
If multiple sso are an option I would let the user choose between Google, Facebook and Fitbit and then the ability to link the other ones. But of we have to decide for them I would probably just use Fitbit because it’s the most related.

Reply:
It would be a bold move to only have one SSO provider, given that the whole point of this project is to remove the dependency of the pebble ecosystem on a single company.

I think it is actually a good idea to drop manual credentials management altogether.

The [official OAuth2](https://github.com/golang/oauth2) package for Go seems to support a wide variety of OAuth providers out of the box (including Fitbit), so I think it would be a good fit.

Reply:
Using multiple should be not substantially more difficult than using just one; instead of just storing a remote-userid as an auth key, you instead store the tuple of a {remote-servicename, remote-userid}.  (Though the fact that there is an official OAuth2 package is vaguely mystifying to me, given that OAuth2 is an incredibly simple protocol...)